docs/data/genomic_harmonization.rst

```diff
@@ -0,0 +1,186 @@
+by the sequencing site.
+The file structure of delivered files is not consistent nor enforced between
+studies and is usually unique to the sequencing center providing the data.
+
+.. Note::
+   Many early studies recieved data in buckets that were provisioned
+   per-sequencing center.
+   Some studies may still have extraneous data scattered in these legacy
+   buckets rather than in the per-study buckets.
+
+
+Project Set Up
+--------------
+
+Studies are set up with at least two projects: one for the final delivered
+data, and one or more for harmonization work to be done in.
+When a study is ready to be harmonized, the delivery project is created under
+the ``kids-first-drc`` Cavatica account and one or more harmonization projects
+are created under the ``kfdrc-harmonization`` user depending on what workflows
+are desired.
+
+A typical set of project ids for a study might look something like:
+
+.. code::
+
+   kfdrc-harmonization/sd-xxxxxxxx-bwa-mem
+   kfdrc-harmonization/sd-xxxxxxxx-gatk-haplotypecaller
+   ...
+   kids-first-drc/sd-xxxxxxxx
+
+Upon setting up a new harmonization project, members of the bioinformatics
+
+
+Mounting Volumes
+----------------
+
+Cavatica allows S3 buckets to be accessed within tasks and workflows by
+mounting them as 'volumes'.
+To provide access to data for harmonization, the study bucket is mounted as
+a volume using read/write permissions in the harmonization projects.
+When harmonized data is to be shared with investigators or the public, the
+bucket is mounted as a volume in the delivery project with read-only
+permissions.
+
+
+Harmonizing
+-----------
+
+The act of harmonizing occurs by having a bioinformatics team member invoke
+a standard pipeline, or set of pipelines, for a number of source inputs in
+a project.
+For example, a bioinformatics team member may write a small script to spawn
+Cavatica tasks running the `bwa-mem` workflow (or `app` in Cavatica terms)
+for every ``.fastq`` file in some ``kfdrc-harmonization/sd-xxxxxxxx-bwa-mem``
+project.
+
+Invoking harmonization tasks involves some manual work as data is often
+provided in various formats requiring effort to sort through and select the
+correct inputs.
+
+.. Note::
+   Historically, harmonization projects may have had multiple workflows
+   within the same project. Future harmonization tasks should be expected
+   to remain segregated between projects specific to the workflow being run.
+
+
+Relocating Harmonized Files
+---------------------------
+
+Harmonization pipelines run in Cavatica have their results output back into
+Cavatica-controlled storage and need to be moved back into Kids First owned
+buckets.
+This occurs when all harmonization tasks have been completed and verified for
+a project by copying the files into the mounted S3 volume and removing the
+original files from Cavatica to save costs.
+The files are copied into prefixes within the mounted study bucket that
+correspond to the pipelines from which the resulted.
```

Comment:
```suggestion
correspond to the pipelines from which they resulted.
```

Reply:
> I think it's dangerous to call all BIX processing "harmonization"

I checked with yuankun and they do consider anything post-alignment as 'analysis' rather than 'harmonization' so I can try to make that distinction.

RE: harmonized-source data (Broad data); I wasn't sure if these nuances were important here. I was more focused on what the data flow was, but I can add in more about 'unharmonized' vs 'harmonized' vs 'analysis'